components/menu/src/Menu.svelte

```diff
@@ -30,15 +31,13 @@
   {#if open}
     <div
       {...$$restProps}
-      class="resp-menu {className}"
+      class="resp-menu resp-menu__{align} {className}"
```

Comment:
Hi @ryantanrk , i think the correct BEM convention would be `resp-menu--{align}`, since align is a variant. Is it possible to change to this way?

Reply:
can